### PR TITLE
Change PhotoGenerator UI into split view for better user experience

### PR DESCRIPTION
**Pull request by dreamingblackcat**
- Changed photo generator layout to 2 column split view
  - Added more controls to photo generator
  - Fixed the bug where user deletes everything in the text box, the text would remain the same on
    the canvas and react keeps re-rendering with error causing page to become unreponsive.
  - Added font embeds from mmwebfont repository (may be we need to add credits in readme as well ?)
  - Added a few css classes for supported myanmar fonts
  - Changed Sketch color picker to Compact picker for getting more space
  - Added opacity as an extra control instead of inside a color picker
  - Added .fuse_hidden files to gitignore (required for ubuntu user devs)
  - And finally, use yarn for packaging (yarn.lock added).

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -14,6 +14,11 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
```
Comment (yelinaung) [inline public/index.html:16]: Can we change the title here?

Comment (dreamingblackcat) [reply public/index.html:16]: Good point! May be change to just `Deepseed` or do you have any touchy line @yelinaung ?

Comment (yelinaung) [reply public/index.html:16]: So Deep So Seed ©

Comment (yelinaung) [reply public/index.html:16]: and the favicon

**Comment by yelinaung** [comment]
@devmtnaing please review asap!

**Review by devmtnaing — COMMENTED** [review]
Everything looks good @dreamingblackcat. 

But, we still need to fix the canvas width since it is going beyond the grid boundary. The rest very solid.

Yay finally another step of deepseed web. (Y)

**Comment by yelinaung** [comment]
[f3df2275b67e847345b7745aff97f20e.ico.zip](https://github.com/teamdeepseed/deepseed-web/files/1343197/f3df2275b67e847345b7745aff97f20e.ico.zip)
@dreamingblackcat favicons

**Pull request closed without merging**